StandIn/StandIn/Program.cs

```diff
@@ -4424,28 +4450,45 @@ static void Main(string[] args)
                                 {
                                     if (ArgOptions.bAdd)
                                     {
-                                        ModifyADCSTemplate(ArgOptions.sFilter, false, true, false, false, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
+                                        ModifyADCSTemplate(ArgOptions.sFilter, false, true, false, false, false, ArgOptions.iSigValue = 0, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
                                     }
                                     else
                                     {
-                                        ModifyADCSTemplate(ArgOptions.sFilter, false, true, false, true, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
+                                        ModifyADCSTemplate(ArgOptions.sFilter, false, true, false, true, false, ArgOptions.iSigValue = 0, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
                                     }
                                 }
                                 else
                                 {
                                     Console.WriteLine("[!] Insufficient arguments provided (--filter/--add/--remove)..");
                                 }
+                            } else if (ArgOptions.bSig)
+                            {
+                                if (!String.IsNullOrEmpty(ArgOptions.sFilter) && ArgOptions.bRemove || (ArgOptions.iSigValue > 0))
+                                {
+                                    if (ArgOptions.iSigValue > 0)
+                                    {
+                                        ModifyADCSTemplate(ArgOptions.sFilter, false, false, false, false, true, ArgOptions.iSigValue, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
+                                    }
+                                    else
+                                    {
+                                        ModifyADCSTemplate(ArgOptions.sFilter, false, false, false, true, false, ArgOptions.iSigValue = 0, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
```

Comment:
4474 Should be:

```suggestion
                                        ModifyADCSTemplate(ArgOptions.sFilter, false, false, false, true, true, ArgOptions.iSigValue = 0, ArgOptions.sDomain, ArgOptions.sUser, ArgOptions.sPass);
```

As it currently is written, bSignature is set to False, so removing the signature flag fails.

Reply:
Nice catch! Thanks for pointing that out, committed the suggestion. Looks good!